rename-desktop-file.patch

```diff
@@ -0,0 +1,25 @@
+From 280a05be1f33b5b2cbc70159a98db24cd98ace81 Mon Sep 17 00:00:00 2001
+From: Alberto Fanjul <albertofanjul@gmail.com>
+Date: Wed, 8 Sep 2021 11:51:31 +0200
+Subject: [PATCH] Rename desktop file
+
+---
+ data/meson.build | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/data/meson.build b/data/meson.build
+index 0e348fd..bb450ec 100644
+--- a/data/meson.build
++++ b/data/meson.build
+@@ -27,7 +27,7 @@ i18n = import('i18n')
+ desktop_file = i18n.merge_file(
+ 	'desktop',
+ 	input: meson.project_name() + '.desktop.in',
+-	output: meson.project_name() + '.desktop',
++	output: 'nl.sarine.gpxViewer.desktop',
```

Comment:
why this patch when you can `rename-desktop-file` ?

Reply:
Exactly, I never was able to remember these details